osu.Game/Overlays/Settings/Sections/SkinSection.cs

```diff
@@ -59,15 +65,32 @@ private void load(OsuConfigManager config, SkinManager skins)
 
             config.BindWith(OsuSetting.Skin, configBindable);
 
+            usableSkins = skins.GetAllUsableSkins().ToArray();
+
             skinDropdown.Bindable = dropdownBindable;
-            skinDropdown.Items = skins.GetAllUsableSkins().ToArray();
+            skinDropdown.Items = usableSkins.Length > 1 ? usableSkins.Concat(new[] { random_skin_info }) : usableSkins;
```

Comment:
I don't really want to call .Add on some array that a function returned because if that function changes to return some existing array instance instead of always creating a new array, it will keep adding that item to the instance